src/main/java/nl/cherement/jak/service/BoardService.java

```diff
@@ -94,14 +104,22 @@ public BoardEntity deleteTeam(BoardEntity boardEntity) {
     }
 
     @Override
-    boolean hasAccess(Authentication authentication, BoardEntity entity) {
-        boolean userAccess = false;
-        for (UserEntity userEntity : entity.users) {
-            if (userEntity.username.equals(authentication.getName())) {
-                userAccess = true;
-                break;
+    boolean hasAccess(Authentication authentication, BoardEntity boardEntity) {
```

Comment:
De `hasAccess` word vaak opgevraagd.
Kunnen we de userservice hier niet uithalen?
En de vergelijkings maken met de username maken met `authentication.getName()` deze is toch uniek.

Reply:
Door de manier waarop hibernate dependencies ophaald hebben we hier niet de usernames van de users op de boards maar alleen de id's. Het is dus de keuze uit elk van deze users ophalen zodat we met username kunnen vergelijken of de requesting user ophalen zodat we met id kunnen vergelijken.